src/main/java/edu/harvard/iq/dataverse/util/FileUtil.java

```diff
@@ -1449,6 +1449,16 @@ public static S3AccessIO getS3AccessForDirectUpload(Dataset dataset) {
     	return s3io;
     }
     
+    private static InputStream getInputStream(StorageIO<DataFile> storage, boolean isTabularData) throws IOException {
```

Comment:
Perhaps call this getOriginalFileInputStream to make it clearer? Also - any reason not to put the storage.open call in this method?

Reply:
Done.